app/ts/store/account.ts

```diff
@@ -57,6 +58,20 @@ async function connect() {
 	}
 }
 
+async function ensureConnected() {
+ 	try {
+ 		assertsExternalProvider(window.ethereum)
+ 		const provider = new ethers.providers.Web3Provider(window.ethereum)
+ 		const signer = provider.getSigner()
+ 		store.value = { status: 'busy' }
+ 		const address = await signer.getAddress()
+ 		store.value = { status: 'connected', address }
+ 	} catch (exception) {
+ 		store.value = { status: 'disconnected', connect, ensureConnected }
```

Comment:
I just realized these are similar values

```suggestion
		store.value = storeDefaults
```